packages/cli/src/databases/entities/workflow-entity.ts

```diff
@@ -91,6 +91,21 @@ export class WorkflowEntity extends WithTimestampsAndStringId implements IWorkfl
 	display() {
 		return `"${this.name}" (ID: ${this.id})`;
 	}
+
+	@Column({ type: 'simple-array', default: '' })
+	credentialIds: string[];
+
+	@Column({ type: 'simple-array', default: '' })
+	nodeTypes: string[];
+
+	@Column({ type: 'simple-array', default: '' })
+	nodeNames: string[];
+
+	@Column({ type: 'simple-array', default: '' })
+	webhookURLs: string[];
+
+	@Column({ type: 'simple-array', default: '' })
+	httpNodeURLs: string[];
```

Comment:
I agree 💯 

I'd also like to look into having our ORM (whichever we end up with) support JSON lookups across sqlite, postgres and MySQL. 

But having the nodes in another table would be even cleaner.

Reply:
irrespective of if we move these to separate tables or not, we should definitely avoid duplicating this data, to avoid having multiple sources of truths.

Reply:
I currently don't have the time to do that. So if somebody wants to take that over, feel free.

Reply:
I can try to have a look in the next days.